profiles/ritika1.md

```diff
@@ -0,0 +1,15 @@
+## Ritika Varshney
+
+### Bio
+I am interseted in Machine learning and web development.
+
+### Location
+Bareilly
+
+
+### Hobbies & Interests
+Listen to music ,cooking.
+
+### Profiles
+[![GitHub](https://github.com/ritikavar) 
```

Comment:
There's a syntax error here. It should be like this:
`[![GitHub][github-img]](https://github.com/ritikavar)`